dataactcore/models/jobModels.py

```diff
@@ -117,10 +117,10 @@ class JobDependency(Base):
     __tablename__ = "job_dependency"
```

Comment:
Missed this last time, should Job also have a cascade on delete or no? Since we're cascading a delete on job_id for a lot of them.